nixos/modules/programs/hyprland.nix

```
@@ -0,0 +1,87 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
+  cfg = config.programs.hyprland;
+
+  defaultHyprlandPackage = pkgs.hyprland.override {
+    enableXWayland = cfg.xwayland.enable;
+    hidpiXWayland = cfg.xwayland.hidpi;
+    nvidiaPatches = cfg.nvidiaPatches;
+  };
+in
+{
+  options.programs.hyprland = {
+    enable = mkEnableOption null // {
+      description = mdDoc ''
+        Hyprland, the dynamic tiling Wayland compositor that doesn't sacrifice on its looks.
+
+        You can manually launch Hyprland by executing "exec Hyprland" on a TTY.
```

```suggestion
        You can manually launch Hyprland by executing {command}`Hyprland` on a TTY.
```

Maybe use the "command" md type here?

---

Right, I forgot about it. Do you maybe have a list of all the md types supported in Nixpkgs? I always forget them.

---

I think the list at https://nixos.org/manual/nixpkgs/unstable/#sec-contributing-markup is exhaustive, but I'm not sure